src/templates/qa_hpwh.mdx

```diff
@@ -51,221 +50,514 @@
-
-Installer: {props.doc.installer}
-
-Install Date: <DateStr date={props.doc.install_date}/>
-
-<Photo id="house_number_photo" label="House Number – Photo" required />
-
-<Photo id="hpwh_location_photo" label="HPWH Location – Photo" required />
-
-<Photo id="adjacent_space_photo" label="Adjacent Space – Photo">
-  This space will supply air to meet the air volume requirement for the HPWH.
-</Photo>
-
-Volume of space supplying air to the HPWH = {props.doc.volume_calculator?.length * props.doc.volume_calculator?.width * props.doc.volume_calculator?.height} ft<sup>3</sup> (Must be at least 700 ft<sup>3</sup>)
-
-<Photo id="completed_installed_photo" label="Completed Installation – Photo" required>
-  This photo shows the completed HPWH installation including the exhaust,
-  insulated hot water line, pressure relief valve discharge, and condensate.
-</Photo>
-
-<Photo id="nameplate_photo" label="Nameplate – Photo" required>
-  This photo shows the nameplate of the installed system including the
-  model number and serial number.
-</Photo>
-
-<Photo id="mode_photo" label="HPWH Screen with Mode – Photo" required>
-  This photo shows the HPWH Screen showing the operation mode selected.
-</Photo>
-
-<Photo id="cta_2045_module_photo" label="CTA-2045 Module - Photo">
-  This photo shows the installed CTA-2045 Module.
-</Photo>
-
-  </PrintSection>
-</Tab>
+    ## Planned Install Area
+    **HPWH Space Evaluation**
+    <Collapsible header="HPWH Space Evaluation — Instructions">
+      The successful installation of a heat pump water heater (HPWH) begins in the planning
+      stage by assessing whether the location where the HPWH will be installed is suitable.
+      The space around the HPWH must be...
+      1. large enough to meet the air volume requirement (700 ft3 minimum) of the HPWH,
+      2. where some compressor and fan noise (50 to 65 dB, normal conversation loudness) is
+      acceptable,
+      3. where some cooled air (approximately a half-ton) is acceptable or desirable,
+      4. not dusty, or where a significant amount of dust is generated,
+      5. tall enough (~12” taller than the current tank height, ~76” if < 65 gals to ~ 86” if >= 80
+      gals) to allow the filter in the HPWH to be periodically accessed for cleaning,
+      6. where there is access to a drain, drain line, or utility sink for condensate, and
+      7. in the 40°–125° F (4.4°–51° C) temperature range year-round
+
+      It is possible to also meet the air volume requirement of the HPWH by installing it next
+      to a louvered door, or ducting the supply, exhaust, or both the supply and the exhaust
+      of the HPWH to an adjacent space. It is also possible to install a condensate pump where
+      a drain located below the elevation of the HPWH’s condensate outlet is not available.
+
+      To verify conditions 1-7 have been met please use the "HPWH Location" photo input to
```

Comment:
This text is inappropriate for the report since it refers to using the photo input.